docker-compose.yml

```diff
@@ -0,0 +1,40 @@
+version: "3.0"
+
+services:
+    front:
+        build:
+            context: ./src/front/
+            dockerfile: ../../dockerfiles/Dockerfile.front
+        ports:
+            - 8080:8080
+        depends_on:
+            - back
+        image: gcr.io/pixie-prod/demos/px-mongo/frontend:0.1.0
```

Comment:
Using the `image:` key as the current latest version of the given container makes it easy to push the current version -- `docker-compose build` followed by a `docker push ${image_name_from_output}`.

It also allows us to easily track what the most recent version pushed for each component. This avoids us having to look at the container registry to pick an available next version (as long as we only push an image once it is merged to the `main` branch).

Can we make sure each of the services that reference a dockerfile has an `image:` set to the future version (in this case `1.0`)? That way once this is merged, we can push the latest and greatest version and it will be our first initial release.